src/prelude.rs

```diff
@@ -91,28 +92,3 @@ impl Default for Spawner {
         }
     }
 }
-
-/// Checks if a collision event contains a bullet. If so, return the entities with the bullet as the first entity
-pub fn is_bullet_collision(event: &CollisionEvent) -> Option<(Entity, Entity)> {
-    is_layer_collision(event, Layer::Bullet)
-}
-
-/// Checks if a collision event contains the specific physics layer. If so, return the entities with the chosen layer as the first entity
-pub fn is_layer_collision(event: &CollisionEvent, layer: Layer) -> Option<(Entity, Entity)> {
-    let entities = event.rigid_body_entities();
-    let layers = event.collision_layers();
-
-    // assert that neither layer is uninitialized
-    assert!(![layers.0, layers.1]
-        .into_iter()
-        .any(|layer| layer.contains_group(Layer::None)));
-
-    match [layers.0, layers.1]
-        .into_iter()
-        .position(|l| l.contains_group(layer))
-    {
-        Some(0) => Some(entities),
-        Some(1) => Some((entities.1, entities.0)),
-        _ => None,
-    }
-}
```

Comment:
I think this helper function may still be useful in the future, I'd prefer to tag as dead code over deleting and having to dig it back up